langchain/scripts/check-tree-shaking.js

```diff
@@ -51,7 +59,12 @@ export async function checkTreeShaking() {
 
     console.log = function (...args) {
       const line = args.length ? args.join(" ") : "";
-      if (line.trim().startsWith("First side effect in")) {
+      const [, moduleWithSideEffects] =
+        line.match(/First side effect in (\S+)/) ?? [];
+      if (
+        moduleWithSideEffects &&
+        !ignoredEntrypoints.has(moduleWithSideEffects)
```

Comment:
I am not sure what the implications of this with bundlers will be - @nfcampos do you have thoughts?

Alternatively could we get this test to pass without this change?

Reply:
Yea this sort of defeats the purpose of this check. We can't make exceptions here

Reply:
Got it, @xixixao are you able to fix?